cmd/soroban-cli/build.rs

```diff
@@ -1,3 +1,45 @@
 fn main() {
     crate_git_revision::init();
+    let w = &mut std::io::stdout();
+    __set_example_contracts(w).unwrap();
+}
+
+fn __set_example_contracts(w: &mut impl std::io::Write) -> std::io::Result<()> {
+    // let example_contracts = get_example_contracts().unwrap().join(",");
+    let example_contracts = get_example_contracts().unwrap().join(",");
```

Comment:
Great idea! Are you thinking something like in [24f80d2](https://github.com/stellar/soroban-cli/pull/1244/commits/24f80d2a0c695fac4fe7dcb45540c260094b54ee)?

Reply:
Yep!

Reply:
Is this worth doing? If you only run it once, then it won't use the cache. And if you run it multiple times, then a cache may result in stale data, and it'll be hard to figure out why (or even notice). Does the release process fetch it over and over, to a point where it's worth caching for a short period of time?

Reply:
While developing, you could have a watch, which would compile and fetch each time a file changed. In this case stale data isn't an issue as it will always happen on a `cargo install` or when built for release. If the data is stale for the local dev, I'm sure they aren't relying on it unless they are developing with it at which case they can delete the cache if needed.